packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -306,6 +306,15 @@ export default function () {
                       addTo(exports, id.node.name, id.node);
                       init.replaceWith(buildExportsAssignment(id.node, init.node).expression);
                       nonHoistedExportNames[id.node.name] = true;
+                    } else if (id.isObjectPattern()) {
+                      for (let i = 0; i < id.node.properties.length; i++) {
+                        const prop = id.node.properties[i];
+                        if (!t.isRestProperty(prop)) {
```

Comment:
@danez Probably yes, but for now: 1) we haven't envs to use it without `transform-object-rest-spread` and 2) seems like we need `babel-plugin-syntax-object-rest-spread` for rest properties: [repl](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=env&targets=Node-7.4&browsers=&builtIns=false&code=export%20const%20%7Bfoo%3A%20test%2C%20...tester%7D%20%3D%20%7B%7D%3B&experimental=true&loose=true&spec=false&playground=false&stage=0)